daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md

```diff
@@ -2064,6 +2065,73 @@ func RequestApprovalActivity(ctx workflow.ActivityContext) (any, error) {
 
 {{< /tabpane >}}
 
+
+## Step 5: Manage Your Workflow
+
+Now that your workflow is running, let's learn how to manage it using the Dapr CLI.
+
+### View Running Workflows
+
+Open a separate terminal and run the following CLI commands.
+
+```bash
+# List all workflows
+dapr workflow list --app-id orderprocessing --connection-string=redis://127.0.0.1:6379
+```
+
+You should see output like:
+
+```
+INSTANCE ID          WORKFLOW NAME              CREATED              LAST UPDATED         RUNTIME STATUS
+order-20240312-001   OrderProcessingWorkflow    2024-03-12 10:00:00  2024-03-12 10:00:05  RUNNING
+```
+
+### Check Workflow History
+
+View the detailed execution history of your workflow:
+
+```bash
+dapr workflow history order-20240312-001 --app-id orderprocessing
+```
+
+### Interact with Your Workflow
+
+#### Raise an External Event
+
+If your workflow is waiting for an external event:
+
+```bash
+dapr workflow raise-event order-20240312-001/ApprovalEvent \
```

Comment:
Can I raise and event like this in this example? I was not sure how this worked.